content/guides/development.md

```diff
@@ -5,182 +5,253 @@ contributors:
   - SpaceK33z
   - rafde
   - fvgs
+  - TheDutchCoder
 ---
 
-On this page we'll explain how to get started with developing and how to choose one of three tools to develop. It is assumed you already have a webpack configuration file.
+T> This guide extends on code examples found in the [`Output Management`](/guides/output-management) guide.
 
-W> Never use any of these tools in production. Ever.
+If you've been following the guides, you should have a solid understanding of some of the webpack basics. Before we continue, let's look into setting up a development environment to make our lives a little easier.
 
+W> The tools in this guide are __only meant for development__, please __avoid__ using them in production!!
 
-## Adjusting Your Text Editor
 
-Some text editors have a "safe write" feature and enable this by default. As a result, saving a file will not always result in a recompile.
+## Using Source Maps
 
-Each editor has a different way of disabling this. For the most common ones:
+When webpack bundles your source code, it can become difficult to track down errors and warnings to their original location. For example, if you bundle three source files (`a.js`, `b.js`, and `c.js`) into one bundle (`bundle.js`) and one of the source files contains an error, the stack trace will simply point to `bundle.js`. This isn't always very helpful as you probably want to know exactly which source file the error came from.
 
-* **Sublime Text 3** - Add `"atomic_save": false` to your user preferences.
-* **IntelliJ** - use search in the preferences to find "safe write" and disable it.
-* **Vim** - add `:set backupcopy=yes` in your settings.
-* **WebStorm** - uncheck `Use "safe write"` in Preferences > Appearance & Behavior > System Settings
+In order to make it easier to track down errors and warnings, JavaScript offers Source Maps, which maps your compiled code back to your original source code. If an error originates from `b.js`, the Source Map will tell you exactly that.
```

Comment:
Maybe link to somewhere else for an intro to source maps and rephrase slightly:

``` md
In order to make it easier to track down errors and warnings, JavaScript offers [source maps](http://blog.teamtreehouse.com/introduction-source-maps), which map your compiled code back to your original source code. If an error originates from `b.js`, the mapping will direct you to that file.
```

I would also lowercase "Source Maps" => "source maps" everywhere (except in titles).